examples/counter/deployments/_test.yaml

```diff
@@ -0,0 +1,46 @@
+---
+name: "Test Deployment"
+network: test
+genesis:
+  wallets:
+    - label: deployer
+      principal: SP3FBR2AGK5H9QBDH3EEN6DF8EK8JY7RX8QJ5SVTE
+      amount: 10
+    - label: wallet-1
+      principal: SP3FBR2AGK5H9QBDH3EEN6DF8EK8JY7RX8QJ5SVTE
+      amount: 20
+  contracts: [bns, pox, costs-v2]
+plan:
+  batches:
+    - id: 0
+      transactions:
+      - emulated-contract-publish:
+          contract: counter
+          emulated-sender: SP3FBR2AGK5H9QBDH3EEN6DF8EK8JY7RX8QJ5SVTE 
```

Comment:
What do you thing about renaming `emulated-sender` to just `sender` ?
If it is part of `emulated-*` I think using `emulated-sender` feels a bit excessive.

Reply:
I'd be strongly in favor of having a `emulated-sender` vs `expected-sender`, so there's no room for ambiguity.